### PR TITLE
Re-validation Slides URL

### PR DESCRIPTION
**Pull request by olegrise**
@andrecardoso please review

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) Validate on blur](https://trello.com/c/oqkRWIuC/109-2-validate-on-blur)

**Comment by olegrise** [comment]
@andrecardoso please review last commit. The changes are:
- check if directive is visible. My original assumption was that `templateComponentSlides` is instantiated every time when <rise-slides> component is selected in the Editor. Turns out that it is instantiated only once when user opens a presentation editor. That's why it's necessary to check if it's visible or not.
- removed check for `$scope.validationError !== 'VALID'`. I realized that we don't save the `'VALID'` state in `$scope.validationError`. Saving it there would make logic a bit more complex, so I decided to remove this optimization (i.e. not revalidating if it's last validation was successful) for the sake of simplicity.

File: web/scripts/template-editor/components/directives/dtv-component-slides.js [hunk 1]
```diff
@@ -58,9 +64,16 @@ angular.module('risevision.template-editor.directives')
...
               $scope.componentId = $scope.factory.selected.id;
               _load();
+              $scope.saveSrc(); //validate Slides URL
             }
           });
 
+          function _directiveIsVisible() {
+            // This directive is instantiated once by templateAttributeEditor
+            // It becomes visible when <rise-slides> is selected
+            return $scope.factory.selected && ($scope.factory.selected.type === 'ise-slides');
```
Comment (andrecardoso) [inline web/scripts/template-editor/components/directives/dtv-component-slides.js]: Is this correct? Shouldn't it be 'rise-slides' ?

Comment (olegrise) [reply web/scripts/template-editor/components/directives/dtv-component-slides.js]: @andrecardoso good catch! I wonder how that happen. Fixed. See the last commit.

**Pull request merged**